Source: http://cvs.fedoraproject.org/viewvc/devel/gettext/
Upstream: has been fixed since last release
Reason: fix wrong macro call

--- gettext-0.17/gettext-runtime/libasprintf/configure.ac	2007-10-07 21:37:36.000000000 +0200
+++ gettext-0.17/gettext-runtime/libasprintf/configure.ac	2010-03-04 23:03:18.468418657 +0100
@@ -54,7 +54,7 @@ dnl Checks for header files.
 dnl Checks for typedefs, structures, and compiler characteristics.
 AC_C_INLINE
 AC_TYPE_SIZE_T
-gl_AC_TYPE_LONG_LONG
+AC_TYPE_LONG_LONG_INT
 gt_TYPE_LONGDOUBLE
 gt_TYPE_WCHAR_T
 gt_TYPE_WINT_T
